ats-mini/Layout-SMeter.cpp

```diff
@@ -87,21 +87,34 @@ static void drawAltStereoIndicator(int x, int y, bool stereo = true)
 static void drawLargeSMeter(int rssi, int strength, int x, int y)
 {
   // S-Meter legend
-  for(int i=x; i<=x+15*16 + 2; i+=2) spr.drawPixel(i, 28+y, TH.scale_line);
   spr.setTextDatum(TC_DATUM);
-  spr.setTextColor(TH.scale_text, TH.bg);
+  spr.setTextColor(TH.scale_text);
 
+  int last_x = x;
   for(int i=0; i<16; i++)
   {
     if(i%2)
     {
-      if(i < 10)  spr.drawNumber(i, x+(i*15)-13, 20+y, 2);
-      if(i == 11) spr.drawString("+20", x+(i*15)-13, 20+y, 2);
-      if(i == 13) spr.drawString("+40", x+(i*15)-13, 20+y, 2);
-      if(i == 15) spr.drawString("+60", x+(i*15)-13, 20+y, 2);
+      int text_width = 0;
+      int text_x = x + (i * 15) - 13;
+      if(i < 10)  text_width = spr.drawNumber(i, text_x, 20+y, 2);
+      if(i == 11) text_width = spr.drawString("+20", text_x, 20+y, 2);
+      if(i == 13) text_width = spr.drawString("+40", text_x, 20+y, 2);
+      if(i == 15) text_width = spr.drawString("+60", text_x, 20+y, 2);
+
+      // Draw the dotted line from end of previous number to start of current number
+      for(int px=last_x; px<text_x-text_width/2; px++) {
+        if(~px & 1) spr.drawPixel(px, 28+y, TH.scale_line);
```

Comment:
Nitpick: invert the condition to get the same result as before (see the attached images, **first:** original behavior, **second:** behavior after these changes, **third:** behavior after inverting the condition).
```suggestion
        if(px & 1) spr.drawPixel(px, 28+y, TH.scale_line);
```

<img width="208" height="74" alt="image" src="https://github.com/user-attachments/assets/8b6aac12-bc90-4bf2-b5bb-21df8d35ca7f" />
<img width="208" height="74" alt="image" src="https://github.com/user-attachments/assets/354d17a5-1409-470e-96c1-ea1293ce7c59" />
<img width="208" height="74" alt="image" src="https://github.com/user-attachments/assets/af1eadc2-7213-4665-a7ee-bea536820ac3" />